controllers/progressivesync_controller_test.go

```diff
@@ -365,6 +365,10 @@ func TestReconcile(t *testing.T) {
 
 		ps := newProgressiveSync("zero-clusters", ns.Name, appSet)
 		ps.Spec.Stages = []syncv1alpha1.Stage{
+			newStage("stage-missing-cluster", 1, 1, metav1.LabelSelector{
```

Comment:
```suggestion
			newStage("stage missing cluster", 1, 1, metav1.LabelSelector{
```
just to be consistent with the other stages